### ISSUE TITLE
Proposal: Survey to gauge our impressions of target audience

### ISSUE THREAD

**Issue by mbonsma** [merely-useful/merely-useful.github.io#60]
## Proposal

1. Some draft personas have already been created (#8), but these require more thought and updating to make sure they reflect our target audience. 
2. To better understand what kinds of people are in our audience, we created a [survey](https://forms.gle/wNAkfAGDXhXK69ie9) to ask members of our writing group what kinds of learners and instructors they have encountered in the past. 

## Background

Personas are idealised characters we create by looking at our consumers, both instructors and learners. By finding out who would be using our product, we can build a picture of those ideal people, and model our work product and marketing towards them.

We'd like to better understand the types of data science learners and instructors you have encountered, their background and learning motivations. We want to avoid developing so many personas that they become unhelpful. We would like to develop a few (perhaps 3 or 4 for each book) personas that cover undergraduate, faculty (who need both teaching materials and training) and non-academics. 

## Pros and Cons

**Pro:**
-   Focus our efforts on the audience we collectively perceive to be most appropriate
-   Avoid creating too many personas that span too diverse a collection of people

**Con:**
-   Delays having personas to work from
-   Another thing for people to fill out
-   Possibly biased to the kinds of people we're already thinking about (defeating the purpose of personas)

## Alternatives

-   Use the personas we already have and add or modify them as we notice gaps.

**Comment by mbonsma** [comment]
8 people have filled out the survey; that should be good for now!

**Comment by mbonsma** [comment]
Here's a summary of the [survey](https://docs.google.com/forms/d/e/1FAIpQLSc1IOUfqco1tAIG418fu2FGTINjrO1opmkHmORG6KnZtiQovg/viewform?usp=sf_link) results.

**TL;DR:** most learners and instructors are from a natural science background and have little experience (learners) or a fair amount of experience (instructors). 

**What we should do with this information:** we could either focus the personas on the most common groups, or we could focus on less common groups with the thinking that these are the groups we won't be thinking of by default anyway.

### 1. Data science learners
Respondents were asked to describe data science learners they have encountered.

#### Discipline background
- Natural science background (i.e. biology, engineering, physics) was the most common by far.
- The second most common was social science, but other disciplines were not far behind (formal science, other, humanities). 
- Non-academic industry background was least common.

#### Experience background
- Some informal experience in learners is most common. No programming experience and some formal programming experience were roughly tied for second place.

#### Motivation for learning
- Applying programming to a specific problem or use case is most common, followed by learning for general interest. Learning to teach is least common.

### 2. Data science instructors
Respondents were asked to describe data science instructors they have encountered.

#### Discipline background
- Natural science background most common, followed by formal science, followed by social science

#### Experience background
- Most common experience level is approximately tied between some formal programming experience and extensive informal programming experience.
- This is a distribution shifted towards more experience than the learners. 

#### Motivation for learning
- All three motivations listed (specific use case, general interest, and learning to teach) were close, but applying programming to a specific use case was the winner, followed by learning to teach.

#### Institutional attitudes
Asked if their institution was encouraging, ambivalent, or discouraging about data science education, 5 respondents said their institution was encouraging and 3 ambivalent.
